File: C program/Basics/array.c
#include<stdio.h>
void main(){
    int size;
    printf("Enter the size : ");
    scanf("%d",&size);
    int arr[size];
    for (int i=0; i<size;i++){
        printf("Enter the value of %d index : ",i);
        scanf("%d",&arr[i]);
    }
    for (int i=0; i<size;i++){
        printf("%d ",arr[i]);
    }
}

File: Java Course/Java/Basics/array.java
import java.util.Scanner;

public class array {
    public static void main(String[] args) {
        Scanner input=new Scanner(System.in);
        System.out.print("Enter the Size of array : ");
        int size = input.nextInt();
        int[] arr = new int[size];
        int sum = 0;
        for (int i=0; i<size; i++){
            System.out.print("enter the "+i+"th index : ");
            arr[i] = input.nextInt();
        }
        for(int j=0; j<size; j++){
            sum+=arr[j];
        }
        System.out.println("Sum : "+sum);
        input.close();
    }
}
